ansible_rulebook/cli.py

```diff
@@ -338,6 +344,10 @@ def main(args: List[str] = None) -> int:
     setup_logging_and_display(args)
 
     if args.controller_url:
+        if not validate_controller_url(args.controller_url):
```

Comment:
@reidliu41 We get the controller URL from the command line and also from the EDA Server via websocket. It might be more helpful if we capture the invalid URL for both the paths this might be a better location for this check https://github.com/ansible/ansible-rulebook/blob/6073d49db7a4981eba7d45b551a42ee25482f58e/ansible_rulebook/app.py#L111

Reply:
updated

Reply:
@mkanoor pls review